auto_dev/protocols/scaffolder.py

```diff
@@ -1,8 +1,13 @@
 """Module for generating protocols from a protocol.yaml specification."""
 
+import re
+import ast as pyast
 import shutil
+import inspect
 import tempfile
+import importlib
```

Comment:
when you see imports of `re` + `ast` + `inspect` + `importlib`, you know 💩 is about to hit the fan